libraries/pbrlib/genglsl/lib/mx_bsdfs.glsl

```diff
@@ -69,7 +69,7 @@ float mx_microfacet_sheen_NDF(float cosTheta, float roughness)
 // LUT for sheen directional albedo. 
 // A 2D table parameterized with 'cosTheta' (cosine of angle to normal) on x-axis and 'roughness' on y-axis.
 #define SHEEN_ALBEDO_TABLE_SIZE 16
-uniform float u_sheenAlbedo[SHEEN_ALBEDO_TABLE_SIZE*SHEEN_ALBEDO_TABLE_SIZE] = float[](
+const float u_sheenAlbedo[SHEEN_ALBEDO_TABLE_SIZE*SHEEN_ALBEDO_TABLE_SIZE] = float[](
```

Comment:
`glslang` errors out on this because it's outside of a uniform block. Recall the same issue we've had in the past with SS and another strict GLSL compiler.